brage-import/src/main/java/no/sikt/nva/brage/migration/mapper/BrageNvaMapper.java

```diff
@@ -70,7 +70,8 @@ public static Publication toNvaPublication(Record brageRecord)
         throws InvalidIssnException, InvalidIsbnException, InvalidUnconfirmedSeriesException {
         var publication = new Publication.Builder().withHandle(extractHandle(brageRecord))
                               .withEntityDescription(extractEntityDescription(brageRecord))
-                              .withCreatedDate(extractPublishedDate(brageRecord))
+                              .withCreatedDate(Instant.now())
```

Comment:
det er ikke datoen på entitydescription som skal settes til instant now. Det er på rot nivået av resource?

I tillegg så bør disse to instansene være like, så du bør lage en variabel og de-referense det slik at create, published og modified blir likt.

Reply:
<img width="367" alt="image" src="https://github.com/BIBSYSDEV/nva-publication-api/assets/44363838/c5c46499-b7cd-4866-ba0f-6d15c819937f">

Reply:
Jeg setter den på rotnivå.
publication.publishedDate og publication.entityDescription.publicationDate er verdiene vi får fra brage.

Reply:
Sorry brain lagger, så ut som du satt det i entitydescription. My bad.